resources/views/volumes/clone.blade.php

```diff
@@ -93,83 +95,76 @@ class="text-muted">#<span v-text="file.id"></span></span> <span
                     </div>
                     <div>
                         <div class="checkbox">
-                            <div v-cloak>
+                            <div>
                                 <label><input type="checkbox" class="checkbox" id="fileLabels"
-                                              v-model="cloneFileLabels" name="clone_file_labels" value="1">
-                                    <span v-text="'Clone '+getFileType(false)+' labels'"></span>
-                                    <div class="form-group{{ $errors->has('clone_file_labels') ? ' has-error' : '' }}">
+                                              v-model="filterFileLabels" name="clone_file_labels" value="1">
+                                    <span>Clone file labels</span>
+                                    <div class="form-group{{ $errors->has('clone_file_labels') ? ' has-error' : '' }}" v-cloak>
                                         @if($errors->has('clone_file_labels'))
                                             <span class="help-block">{{ $errors->first('clone_file_labels') }}</span>
-                                        @else
-                                            <span class="help-block"
-                                                  v-text="'Check, if '+getFileType(false)+' labels should be cloned too'">
-                                    </span>
                                         @endif
                                     </div>
                                 </label>
                             </div>
 
-                            <div v-if="cloneFileLabels" v-cloak>
+                            <div v-if="filterFileLabels" v-cloak>
                                 <label><input type="checkbox" class="checkbox" id="restrictFileLabels"
                                               v-model="restrictFileLabels">
-                                    <span v-text="'Restrict '+getFileType(false)+' labels '+'('+selectedFileLabelsCount+' labels selected)'"></span>
-                                    <div class="form-group{{ $errors->has('only_file_labels') ? ' has-error' : '' }}">
+                                    <span
+                                        v-text="'Restrict file labels '+'('+selectedFileLabelsCount+' labels selected)'"></span>
```

Comment:
This should become "filter file labels", right? Also, an alternative implementation could be:

```html
Filter file labels (<span v-text="selectedFileLabelsCount"></span> labels selected)
```